internal/datastore/postgres/common/pgx.go

```diff
@@ -109,6 +107,56 @@ func ConfigurePGXLogger(connConfig *pgx.ConnConfig) {
 	addTracer(connConfig, &tracelog.TraceLog{Logger: levelMappingFn(l), LogLevel: tracelog.LogLevelInfo})
 }
 
+// truncateLargeSQL takes arguments of a SQL statement provided via pgx's tracelog.LoggerFunc and
+// truncates SQL statements and SQL arguments to placeholders that exceed a certain length. This helps
```

Comment:
truncateLargeSQL takes arguments of a SQL statement provided via pgx's tracelog.LoggerFunc and
replaces SQL statements and SQL arguments with placeholders when the statements and/or arguments exceed a certain length